app/src/main/java/io/github/droidkaigi/confsched2018/data/db/FavoriteFirestoreDatabase.kt

```diff
@@ -169,3 +178,8 @@ class FavoriteFirestoreDatabase : FavoriteDatabase {
         private const val DEBUG: Boolean = false
     }
 }
+
+fun <R> Task<R>.toSingle() = Single.create<R> { emitter ->
+    this.addOnSuccessListener { emitter.onSuccess(it) }
```

Comment:
👍

Reply:
Sorry, I don't know where to put the extension, so put in same file.

Reply:
If you think this extension method is useful for other files, Please put it FirebaseTaskExt.kt in util.ext package

Reply:
Thanks! I'll do it.